From 461dc4ca872686f127b25df970f8fb9e4e4b9de9 Mon Sep 17 00:00:00 2001
From: Jonathan Piat <piat.jonathan@gmail.com>
Date: Sun, 1 Mar 2015 18:48:19 +0100
Subject: [PATCH] Adding LOGI-Bone R1 to cape list

---
 arch/arm/boot/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index f64c1cd..c82824f 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -258,6 +258,7 @@ dtb-$(CONFIG_ARCH_OMAP2PLUS) += omap2420-h4.dtb \
 	am335x-bone-lcd7-01-00a2.dtb \
 	am335x-bone-lcd4-01-00a1.dtb \
 	am335x-bone-lcd3-01-00a2.dtb \
+	am335x-bone-logibone-r1.dtb \
 	am335x-bone-bbb-exp-c.dtb \
 	am335x-bone-4dcape-70t.dtb \
 	am335x-bone-4dcape-70.dtb \
-- 
1.9.1

